metatron-app/metatron-discovery
# F#1071 dataset has no projection

### Description

Removed projection from dataset related APIs

**Related Issue** : 

#1071

### How Has This Been Tested?

1. Make one dataset -> Go to dataset detail -> if detail page works well with no error IT's OK

2. Go to dataset detail -> try `Add to exisiting dataflow` & `Create dataflow with this dataset` -> if both works well with no error -> IT's OK

3 . Go to dataflow detail -> click wrangled dataset -> check if you can see the rule list in the dataflow panel on the right 


### Types of changes



### Checklist:



### Additional Context
